doc/source/index.rst

```diff
@@ -4,16 +4,25 @@
 Welcome to the documentation of SpiceEV!
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
```

Comment:
Full name "Simulation Program for Individual Charging Events of Electric Vehicles" should be given here